src/watertap_contrib/seto/costing/units/chemical_softening_zo.py

```diff
@@ -0,0 +1,968 @@
+        mutable=True,
+        doc="Constant in the flocculation tank capital cost equation",
+    )
+
+    blk.sed_basin_op_coeff_1 = Param(
+        initialize=7e-10,
+        mutable=True,
+        doc="Coefficient of first term in sedimentation basin operational cost equation",
+    )
+
+    blk.sed_basin_op_coeff_2 = Param(
+        initialize=-0.00005,
+        mutable=True,
+        doc="Coefficient of second term in sedimentation basin operational cost equation",
+    )
+
+    blk.sed_basin_op_coeff_3 = Param(
+        initialize=1.5908,
+        mutable=True,
+        doc="Coefficient of second term in sedimentation basin operational cost equation",
+    )
+
+    blk.sed_basin_op_exp_1 = Param(
+        initialize=3,
+        mutable=True,
+        doc="Exponent of the first term in the sedimentation basin operational cost equation",
+    )
+
+    blk.sed_basin_op_exp_2 = Param(
+        initialize=2,
+        mutable=True,
+        doc="Exponent of the first term in the sedimentation basin operational cost equation",
+    )
+
+    blk.sed_basin_op_constant = Param(
+        initialize=6872,
+        mutable=True,
+        doc="Constant in the sedimentation basin operational cost equation",
+    )
+
+    blk.recarb_basin_op_coeff_1 = Param(
+        initialize=1e-8,
+        mutable=True,
+        doc="Coefficient of first term in recarbonation basin operational cost equation",
+    )
+
+    blk.recarb_basin_op_coeff_2 = Param(
+        initialize=-0.0004,
+        mutable=True,
+        doc="Coefficient of second term in recarbonation basin operational cost equation",
+    )
+
+    blk.recarb_basin_op_coeff_3 = Param(
+        initialize=6.19,
+        mutable=True,
+        doc="Coefficient of third term in recarbonation basin operational cost equation",
+    )
+
+    blk.recarb_basin_op_exp_1 = Param(
+        initialize=3,
+        mutable=True,
+        doc="Exponent of first term in recarbonation basin operational cost equation",
+    )
+
+    blk.recarb_basin_op_exp_2 = Param(
+        initialize=2,
+        mutable=True,
+        doc="Exponent of second term in recarbonation basin operational cost equation",
+    )
+
+    blk.recarb_basin_op_constant = Param(
+        initialize=10265,
+        mutable=True,
+        doc="Constant in recarbonation basin operational cost equation",
+    )
+
+    blk.lime_feed_system_op_coeff = Param(
+        initialize=4616.7,
+        mutable=True,
+        doc="Coefficient in lime feed system operational cost",
+    )
+
+    blk.lime_feed_system_op_exp = Param(
+        initialize=0.4589,
+        mutable=True,
+        doc="Constant in lime feed system operational cost",
+    )
+
+    blk.admin_op_coeff = Param(
+        initialize=88589,
+        mutable=True,
+        doc="Coefficient in lime feed system operational cost",
+    )
+
+    blk.admin_op_exp = Param(
+        initialize=0.4589,
+        mutable=True,
+        doc="Exponent in lime feed system operational cost",
+    )
+
```

Comment:
Can you imagine a case where you might (in the future) want to fit some of these parameter values based on some provided set of data? If yes, then those params should be vars (and subsequently fixed). If not, keep as is. For parameter estimation, parameters have to be declared as variables to make use of IDAES' parameter estimation tool.

Reply:
My 2cents - I don't think these would be worth using parmest for or that you'd have a situation where you couldn't fit it using Excel or something else and then just change the mutable parameter values. The regression approach would also have to match what the reference used here, and I imagine that might not be the case. Obviously it is easy to change them to vars and they are fixed via the ``@register_costing_parameter_block`` decorator, so the effort is minimal. But I am agnostic on the utility I guess.